lib/sles4sap/ipaddr2.pm

```diff
@@ -439,7 +439,7 @@ sub ipaddr2_infra_deploy {
         my $wt = az_vm_wait_running(
             resource_group => $rg,
             name => ipaddr2_get_internal_vm_name(id => $_),
-            timeout => 600
+            timeout => 1200
```

Comment:
Double the value we doubled in https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/22264